cfglib.py

```diff
@@ -279,7 +281,8 @@
                help='name of pool for volumes in Ceph RBD storage'),
     cfg.StrOpt('snapshot_name_template', default='snapshot-',
                help='template for creating names of snapshots '
-                    'on storage backend')
+                    'on storage backend'),
+    cfg.StrOpt('nfs_share', help='NFS share address'),
```

Comment:
Let's make this more readable, let's call option `nfs_volumes_remote_target`, and `help="Remote target for NFS shares used for cinder volumes. Must be used with `[migrate] cinder_migration_strategy = 'cloudferrylib.os.storage.cinder_database.CinderStorage'``